.changelog/15817.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:feature
```

Comment:
```suggestion
```release-note:breaking-change
```

Reply:
It's probably also work adding something in the upgrade guide about this https://github.com/hashicorp/consul/blob/main/website/content/docs/upgrading/upgrade-specific.mdx#consul-115x